src/seedu/addressbook/AddressBook.java

```diff
@@ -1153,6 +1153,15 @@ private static String getUsageInfoForExitCommand() {
     private static String removePrefixSign(String s, String sign) {
         return s.replace(sign, "");
     }
+    
+    /**
+     * Removes prefix in fullString
+     */
+    private static String removePrefix(String fullString, String prefix) {
+    	if (fullString.indexOf(prefix) == 0)
```

Comment:
Java coding stands require you to have braces for if clauses, even if its once line. It prevents issues when future developers edit your code.